bot.py

```diff
@@ -467,6 +497,58 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def show_help(self, message, _command_content):
```

Comment:
I still think a main help message should exist.

If we have a main help message and nice docs in the `README.md` or wiki, do we need per-command help messages?

Calling all maintainers!
@michael-je  @thorduragust

Reply:
I think that's a great idea gardarandri. We could a main help message that would display at the top giving a summary of the bot and it's functionality, then follow that up with a list of all commands and a short explanation (which would be programatic). We could also put a link to the docs in the main help message.

I think that adding a per-command help message would also be nice via `!help play` or `!help !play`, for example. But I don't think this would see much use and therefor should be a high priority.

Reply:
Updated help messages.